packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1914,6 +1914,12 @@ export class ContainerRuntime
 	}
 
 	public processSignal(message: ISignalMessage, local: boolean) {
+		const shouldDisableSignalForSummarizer =
```

Comment:
I'd suggest caching this value in ctor and not asking config every time.
I'd also suggest to revert FG and use it as a kill-bit switch (in case something goes wrong), but enable this change by default.
That all said, it feels like we should be symmetrical here - if we disallow this client to receive signals, it should also not send any signals. Sending ideally should result in exception (and logging statement), but it's much more invasive change, so maybe we should start with just logging on sending path.

Do you think it's possible to move these checks to container runtime? This feels like much better place to do so. To some extend, we deal with custom runtime implementation - summarizer runtime (that's mental model I have here)